src/ReactiveUI/Platforms/windows-common/ReactiveUserControl.cs

```diff
@@ -36,6 +36,33 @@ namespace ReactiveUI
     /// ]]>
     /// </code>
     /// </para>
+    /// <para>
+    /// Note that UWP projects do not support the <c>TypeArguments</c> attribute. The XAML designer window in WPF projects also do not
+    /// support generic types. To use <see cref="ReactiveUserControl{TViewModel}"/> in XAML documents you need to create a base class
```

Comment:
You don't have to do it this way. The xaml designer is do fragile I gave up on it years ago. So might be worth saying more like "to use the base class"

Reply:
It doesn't compile in an UWP (it does in WPF) project when you use ReactiveUserControl. The xaml designer is an extra for me.